### ISSUE TITLE
Use of const in strict mode

### ISSUE THREAD

**Issue by Jester98x** [colindembovsky/cols-agent-tasks#42]
Trying to run the replaceToken task in TFS 2015 (on prem) I get this:

2017-01-18T13:56:01.3148978Z const tl = require('vsts-task-lib/task');
2017-01-18T13:56:01.3148978Z ^^^^^
2017-01-18T13:56:01.3148978Z SyntaxError: Use of const in strict mode.

Build server is running Node:
\> node -v
v6.9.4

Am I missing something? Known issue? Anything else I can add to assist with debugging?

Thanks

**Comment by colindembovsky** [comment]
Hi @Jester98x - haven't seen this before. Which version of TFS are you on? Also, can you set `system.debug` to `true` in the build/release variables so that the build produces verbose logs and then attach the (sanitized if necessary) log?

**Comment by Lanthir** [comment]
I was receiving the same error. I am using TFS 2015 Update 2 with Node v6.9.4. Every search I tried regarding use of const in strict mode pointed me to Node v0.12 or earlier. Thanks to @colindembovsky for suggesting system.debug=true. I found that the build agent is running Node from <agentpath>\Agent\Worker\Handlers\Node and not my installed version. The version from Handlers is v0.12. I copied Node.exe from my installation and it's working now.
